.github/workflows/nix-comment.yml

```diff
@@ -0,0 +1,58 @@
+name: "Nix comment bot posting commands for each PR"
+on: pull_request_target
+jobs:
+  nix-command-pr-comments:
+    permissions:
+      pull-requests: write
+      issues: write
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    name: "Nix command PR comments"
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+      - name: Comment PR
+        uses: "./.github/templates/comment"
+        with:
+          message: |
+            ### Nix commands for this PR
+            **NOTE: You can also run our Nix commands in Docker. See the bottom of this comment.**
+            #### Make sure you have setup the Composable community cache:
+            _(you only need to run it once on your machine)_
+            ```bash
+            nix-shell -p cachix --command "cachix use composable-community"
+            ```
+            #### Show all possible apps, shells and packages:
+            ```
+            nix flake show "github:ComposableFi/composable/${{ github.event.pull_request.head.sha }} --allow-import-from-derivation
+            ```
+            #### Run the Composable node alone:
+            ```bash
+            nix run "github:ComposableFi/composable/${{ github.event.pull_request.head.sha }}#composable-node" -L
+            ```
+            #### Spin up a local devnet:
+            ```bash
+            nix run "github:ComposableFi/composable/${{ github.event.pull_request.head.sha }}#devnet" -L --option sandbox relaxed --show-trace
+            ```
+            #### Spin up a local XCVM devnet:
+            ```bash
+            nix run "github:ComposableFi/composable/${{ github.event.pull_request.head.sha }}#devnet-xcvm" -L
+            ```
+            ### View the docs:
+            ```bash
+            nix run ".#docs-server"
+            ```
+            ### Run this without Nix in Docker.
+            ```bash
+            docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v nix:/nix -it nixos/nix bash -c "nix-env -iA nixpkgs.cachix && cachix use composable-community && nix run github:ComposableFi/Composable/${{ github.event.pull_request.head.sha }}#devnet-dali -L --extra-experimental-features nix-command --extra-experimental-features flakes"
+            ```
+            _NOTE: You can swap `devnet-dali` in the command above with any Nix package_
+            _For more info on how to use Nix, check out our [Nix docs](https://docs.composable.finance/nix.html)_
```

Comment:
Since composable is now in the flake registry; can we simplify the commands?

Reply:
Actually we only want to use the flake registry entry for commands that run on branch `main`, since almost all PRs will be made from forks. In order to accoomodate forks, I've changed the bot to use `${{ github.event.pull_request.repository}}` instead

Reply:
We could use the `composable` registry entry for PRs that originate from the monorepo instead of from a fork, but I don't think that's worth it since it should happen very infrequently and it complicates this job